src/brasil/gov/vcge/browser/viewlets.py

```diff
@@ -11,21 +11,21 @@
 
 
 class VCGEViewlet(ViewletBase):
-    ''' Viewlet adicionado a estrutura visual do portal
-    '''
+    """ Viewlet adicionado a estrutura visual do portal
```

Comment:
one-liner docstrings must be written in one line only: `"""Viewlet adicionado a estrutura visual do portal."""`

https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
Will open an issue to change this in all packages in the foreseeable future. https://github.com/plone/plone.recipe.codeanalysis/issues/76#issuecomment-98839883